contrib/ansible/openshift/README.md

```diff
@@ -28,10 +30,13 @@ Your inventory should have groups of hosts describing the roles they play in the
 
 [lb]
 
+[glusterfs]
+
 [pbench-controller:vars]
 register_all_nodes=False
 
-By default, tools registration is done on only two of the nodes, setting the register_all_nodes to True will register tools on all of the nodes.
+Note: glusterfs group represents cns nodes.
```

Comment:
@ndokos the inventory file used by openshift-ansible will have the cns nodes under glusterfs group.